tests/taproot_swap.rs

```diff
@@ -305,25 +176,32 @@ fn verify_taproot_swap_results(
             i, balances.regular, balances.swap, balances.contract, balances.fidelity, balances.spendable
         );
 
-        // In taproot swaps, makers sweep their incoming contracts
-        // Swept coins are tracked as SweptCoinV2 and appear in swap balance
         // Use spendable (regular + swap) for comparison
-        let max_fees = Amount::from_sat(100000); // Maximum expected mining fees
-
-        // The maker's spendable balance should be at least (original - max_fees + min_earned_fees)
-        let expected_minimum = original_spendable - max_fees;
-        assert!(
-            balances.spendable >= expected_minimum,
-            "Taproot Maker {} balance should not decrease significantly. Original spendable: {}, Current spendable: {}, Expected minimum: {}",
-            i,
-            original_spendable,
-            balances.spendable,
-            expected_minimum
+        assert_in_range!(
+            balances.spendable.to_sat(),
+            [15020199, 15031704], // here no fund loss because swap never happen
```

Comment:
"swap never happens"? Balances did increase. The comment should be something like "makers have more than 15 million sats as they earned swap fees".